src/test/java/org/broadinstitute/consent/http/db/LibraryCardDAOTest.java

```diff
@@ -134,7 +225,7 @@ void testFindLibraryCardByInstitutionId() {
     assertNotNull(cardsFromDAO);
     assertEquals(cardsFromDAO.size(), 1);
     assertEquals(cardsFromDAO.get(0).getId(), libraryCard.getId());
-    assertNull(cardsFromDAO.get(0).getDaaIds());
+    assertEquals(cardsFromDAO.get(0).getDaaIds().size(),0);
```

Comment:
An alternative is to check for list emptiness here. Not a required change, but depending on the circumstances, it could be cleaner.